src/knex-tables.d.ts

```diff
@@ -47,8 +47,8 @@ declare module "knex/types/tables" {
     dependencies?: string[];
     allow_list?: string[];
     wait_list?: string[];
-    added_at?: Date;
-    modified_at?: Date;
+    created_at: Date;
+    modified_at: Date;
```

Comment:
Not opposed to changing this, but do we need to have a db migration for this then?

Reply:
No, the col is already called `created_at` in our db.

I'm not sure why it was `added_at` in this type file.. It was wrong